src/components/Home.js

```diff
@@ -0,0 +1,15 @@
+const Home = () => (
+  <div className="home">
+    <h1>Welcome to Bookstore</h1>
+    <ul>
+      <li>bookname - author</li>
+    </ul>
+    <form>
+      <input type="text" placeholder="title" />
+      <input type="text" placeholder="author" />
+      <button type="submit">Add Book</button>
+    </form>
+  </div>
+);
```

Comment:
- [ ] **Good job so far 👏 👏** However, It would be better to create a component called `Book` for displaying a single book and receiving the *title* and *author* as Props, 😀😎
- [ ] Each book should have a button for removing it (no functionality yet). and the created component should be reused to display a list of books. 👍

Reply:
@TuyishimireEric Great suggestions 👍